ProboCI/probo
# Feature/lol/add generic lamp task

This succeeds and fixes https://github.com/ProboCI/probo/pull/26

The earlier PR contributes the following:
 - Create a generic LAMP plugin.
 - Creates a wordpress plugin which is extended from the LAMP plugin.
 - Reworks the Drupal plugin to extend the LAMP plugin.

In addition to the work performed in the related PR the following additions are made here: 
 - Fixes tests related to the LAMP plugin.
 - Fixes issue in which mysql database did not properly import when using the LAMP plugin.
 - Removes BUILD_DOMAIN export as this was removed upstream and prevents the container manager from creating containers.


### To test
 - Create repositories for each of the following.
    - Generic LAMP application - similar to https://github.com/lliss/probo-php-experiments
    - Wordpress similar to https://github.com/lliss/wordpress-test
    - Drupal similar to https://github.com/lliss/drupal-bear-test
 - Link each repo via the probo UI.
 - Upload a database asset for each PR.
 - Create PRs for each repo.
 - Ensure the database is properly created on the docker container and the build steps have been executed to create a working site.
